ScamSearcher/dev_settings.py

```diff
@@ -0,0 +1,15 @@
+"""This is additional settings config for development only."""
+from ScamSearcher.settings import *
```

Comment:
__[flake8]__
* [F403] 'from ScamSearcher.settings import *' used; unable to detect undefined names